enhancements/machine-config/manage-boot-images.md

```diff
@@ -490,6 +506,28 @@ This is a solution aimed at reducing usage of outdated artifacts and should not
 How will UX be reviewed and by whom? TBD 
 The UX element involved include the user opt-in and opt-out, which is currently up for debate. 
 
+### Enforcement of bootimage skew
+
+There should be some mechanism that will alert the user and fail if an unsupported bootimage is used. For example, the first boot pivot should fail if a RHEL8->RHEL10 pivot is attempted.
+
+#### Acceptable skew
+
+The release payload will describe the current skew policy. Generally speaking, we would like to keep the bootimage version aligned to the RHEL version we are shipping in the payload. For example, a 9.6 bootimage will be allowed until 9.8 is shipped via RHCOS. We would like to keep this customizeable, such that any major breaking changes outside of RHEL major/minor can still be enforced as a one-off.
+
+#### Enforcement options
+
+Some combination of the following mechanisms should be implemented to alert users, particularly non-machineset backed scaled environments. The options generally fall under proactive enforcement (require users to updated and acknowledge upon upgrading to a new version) vs. reactive enforcement (only fail when a non-compliant bootimage is being used to scale into the cluster).
```

Comment:
"require users to updated and acknowledge upon upgrading to a new version" is still reactive, isn't it?  The admin ack approach is proactive, so the admin is aware (and ideally can set things up ahead of time), before updating from 4.y to a 4.(y+1) that would require a newer boot image.